src/storage/src/region/tests/compact.rs

```diff
@@ -220,6 +226,42 @@ impl CompactionTester {
 
         self.object_store.remove_all("/").await.unwrap();
     }
+
+    async fn reopen(&mut self) -> Result<bool> {
+        // Close the old region.
+        if let Some(base) = self.base.take() {
+            base.close().await;
+        }
+
+        // Reopen the region.
+        let object_store = new_object_store(&self.store_dir, None);
```

Comment:
Is it possible to reuse other helper function to construct the region?

Reply:
Looks like it's not used by other cases, so I did not extract it to a new function.